backend/spec/services/checkin/updater_spec.rb

```diff
@@ -190,7 +190,8 @@
         end
       end
       context "on a past checkin" do
-        before { checkin.update!(date: Time.zone.today - 1.day) }
+        let(:checkin) { create(:checkin, user_id: user.id, date: Time.zone.today - 1.day) }
```

Comment:
`update!` in the `before {}` caused an lazy loading error (from Bullet), use a lazy let instead.